include/picongpu/particles/pusher/particlePusherAcceleration.hpp

```diff
@@ -47,8 +47,8 @@ struct Push
     /* this is an optional extension for sub-sampling pushes that enables grid to particle interpolation
      * for particle positions outside the super cell in one push
      */
-    typedef typename pmacc::math::CT::make_Int<simDim,0>::type LowerMargin;
-    typedef typename pmacc::math::CT::make_Int<simDim,0>::type UpperMargin;
+    using LowerMargin =  pmacc::math::CT::make_Int<simDim,0>::type;
```

Comment:
Please remove the second space after `=`

Reply:
fixed and squashed